vendor/modules.txt

```diff
@@ -205,13 +205,13 @@ github.com/hashicorp/terraform-plugin-framework/provider/schema
 github.com/hashicorp/terraform-plugin-framework/providerserver
 github.com/hashicorp/terraform-plugin-framework/resource
 github.com/hashicorp/terraform-plugin-framework/resource/schema
+github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault
```

Comment:
It seems like this is not up to date

Reply:
Sorry ya I had the change to remove this package again local.